WordPress/src/main/java/org/wordpress/android/ui/accounts/login/Login2FaFragment.java

```diff
@@ -138,7 +146,9 @@ public static Login2FaFragment newInstanceSocialConnect(String emailAddress, Str
 
     @Override
     protected void setupLabel(TextView label) {
-        // nothing special to do, just leave the string setup via the xml layout file
+        label.setText(sentSmsCode ? getString(R.string.enter_verification_code_sms, mPhoneNumber)
```

Comment:
can we null check here?

Reply:
Good idea.  Since there are multiple instances of the `label.setText` statement in the code, I thought it might be better to add the `@NonNull` annotation to avoid `NullPointException`.  I added the annotation to the abstract method and instances in 2e2c3d3da9efa0d92557e51664d492196b9d16a3.

Reply:
👍 thanks!